test/csv/csv-filter/test_csv_filter.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: false
+
+require_relative "../helper"
+
+class TestFilter < Test::Unit::TestCase
+
+  def setup
+    @input = [
+      %w[aaa bbb ccc].join(","),
+      %w[ddd eee fff].join(","),
+      "" # Force trailing newline.
+    ].join("\n")
+  end
+
+  # Return filepath of file containing CSV data.
+  def csv_filepath(input, dirpath, option)
+    filename = "#{option}.csv"
+    filepath = File.join(dirpath, filename)
+    File.write(filepath, input)
+    filepath
+  end
+
+  # Return stdout and stderr from CLI execution.
+  def run_csv_filter(filepath, *cli_option_names)
```

Comment:
Can we simplify argument name?

```suggestion
  def run_csv_filter(filepath, *options)
```